docs/includes/quickstart-vs-prerequisites.md

```diff
@@ -3,7 +3,7 @@
     > [!NOTE]
     > If you've previously installed Visual Studio 2017, [use the Visual Studio Installer](https://docs.microsoft.com/visualstudio/install/modify-visual-studio) to ensure that the **Office/SharePoint development** workload is installed. 
 
-- Office 2016 
+- Office 
     
     > [!NOTE]
     > If you don't already have Office 2016, you can [register for a free 1-month trial](http://office.microsoft.com/try/?WT%2Eintid1=ODC%5FENUS%5FFX101785584%5FXT104056786).
```

Comment:
This says Office 2016 - maybe "latest version of Office"?

Reply:
Fixed.